src/ocamlorg_web/lib/redirection.ml

```diff
@@ -211,6 +211,51 @@ let t =
   ; "/index.html", Url.index
   ; "/index.fr.html", Url.index
   ; "/contributors.html", Url.index
+  ; "/ocamllabs/index.html", "/???"
```

Comment:
I don't think these are necessary

Reply:
These were fetched from the main branch. I am currently trying to reference the PR which added them.

Reply:
I'm not sure what happened, but if they show in the diff, it's because these are changes when compared from the main branch, so you can delete them 🙂

Reply:
I am raising a new PR without them.

Reply:
@tmattio, Can you please rerun the checks on this PR? I don't understand why they are failing.